websites/A/Animevost/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "denfake2",
+		"id": "870352885089730561"
+	},
+	"service": "Animevost",
+	"description": {
+		"en": "The original site animevost (animevost), where you can watch uninterrupted anime online (anime online) in good quality, without advertising, stable, and without delay. Fastest new anime series dubbing an hour after release in Japan.",
+		"ru": "Оригинальный сайт animevost (анимевост), на котором вы можете смотреть бесперебойно аниме онлайн (anime online) в хорошем качестве, без рекламы, стабильно и без задержке. Самая быстрая озвучка новых серий аниме через час после выхода в японии."
```

Comment:
```suggestion
		"en": "Base #1 on watching anime online for free.",
		"ru": "База №1 по просмотру аниме онлайн бесплатно."
```

Reply:
Where did you get this from @Syjalo?

Reply:
<details>

![image](https://user-images.githubusercontent.com/62868468/170868288-f1a7ba08-f880-464d-ac9f-b4640052c1c7.png)

</details>

Reply:
No one told you to edit the english description, the old one is correct and this change should be reverted